record-and-playback/core/scripts/bigbluebutton.yml

```diff
@@ -19,6 +19,17 @@ redis_port: 6379
 # Uncomment and set password if redis require it.
 # redis_password: changeme
 
+# redis_workers_host: 127.0.0.1
+# redis_workers_port: 6379
+
+# sequence of recording steps
+steps:
+  archive: "sanity"
+  sanity:
+    - "captions"
```

Comment:
I'm not sure how exactly the syntax works here?

The "captions" step has to finish before you can start the "process" step, so it might have to be added as a separate step.

Reply:
Ok, I didn't know the `process` step depended on the `captions`. I think we just have to change it to the block below and it will work:

```
steps:
  archive: "sanity"
  sanity: "captions"
  captions: "process:presentation"
  process:
    presentation: "publish:presentation"

Reply:
Pushed some improvements to this in 95573b2467037fae8cfe661e66ae507ffe4bc04e